Source/WebKit/UIProcess/API/Cocoa/WKWebViewPrivate.h

```diff
@@ -633,6 +634,8 @@ typedef NS_OPTIONS(NSUInteger, WKDisplayCaptureSurfaces) {
 - (void)_isNavigatingToAppBoundDomain:(void(^)(BOOL))completionHandler WK_API_AVAILABLE(ios(14.0));
 - (void)_isForcedIntoAppBoundMode:(void(^)(BOOL))completionHandler WK_API_AVAILABLE(ios(14.0));
 
+- (UITextInputTraits *)textInputTraits WK_API_AVAILABLE(ios(WK_IOS_TBA));
```

Comment:
shouldn't this be `_textInputTraits` ?

Reply:
Right the idea was to conform to `UITextInputTraitsLike` – I didn't explicitly declare conformance, since it would be little messy (would need to use a `__has_include` since it's a private protocol in a different framework, being declared in our private header).

Any preference on `_` vs. not? Maybe we should add the `_` because this is technically SPI?

Reply:
> Right the idea was to conform to `UITextInputTraitsLike` – I didn't explicitly declare conformance, since it would be little messy (would need to use a `__has_include` since it's a private protocol in a different framework, being declared in our private header).
> 
> Any preference on `_` vs. not? Maybe we should add the `_` because this is technically SPI?

Makes sense — up to you! Adding the `_` to make it explicitly SPI seems sensible.

Reply:
`_` it is!